File: jacivan033/atividade_avaliativa1/questao10.py
print("Programa leitor de segundos ")
h = int(input("Informe a quantidade de horas:"))
m = int(input("Informe a quantidade de minutos:"))
s = int(input("informe a quantidade de segundos:"))
d = h * 3600
a = m * 60
q = a + d + s
print("Os segundos são: " + str(q))

File: jacivan033/atividade_avaliativa1/questao10.java
import java.util.Scanner;

public class questao10 {
    public static void main (String[] args) {
        int h;
        int m;
        int x;
        int r;
        Scanner s;

        s = new Scanner(System.in);
        System.out.println("**Conversor de minutos**");

        System.out.println("Informe a qtd de horas:");
        h = s.nextInt();
        System.out.println("Informe a qtd de minutos:");
        m = s.nextInt();
        System.out.println("Informe a qtd de segundos:");
        x = s.nextInt();

        r = h * 60 + 60 * m + x;

        System.out.println( );
        System.out.println("O resultado eh: " + r);
    }
}               
